app/packages/multimodal/src/views/episode/state-action/StateActionTile.module.css

```
@@ -0,0 +1,451 @@
+}
+
+.retryButton:hover {
+  background: color-mix(
+    in srgb,
+    var(--color-content-text-secondary) 12%,
+    transparent
+  );
+}
+
+.srOnly {
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)